test/rust_src/src/fns-tests.el

```diff
@@ -85,6 +85,12 @@
     (should (equal [nil nil nil nil nil t t t t t] (vconcat (reverse A))))
     (should (equal A (reverse (reverse A))))))
 
+(ert-deftest test-reverse-bool-vector ()
```

Comment:
This seems like an odd name for this test, and it duplicates an existing name :)

Reply:
Yes. I wonder what I was thinking when I came up with the name.